cmd/kubeadm/app/cmd/phases/join/controlplanejoin.go

```diff
@@ -29,14 +29,6 @@ import (
 	etcdphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/etcd"
 	markcontrolplanephase "k8s.io/kubernetes/cmd/kubeadm/app/phases/markcontrolplane"
 	uploadconfigphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/uploadconfig"
-	"k8s.io/kubernetes/pkg/util/normalizer"
-)
-
-var (
-	controlPlaneJoinExample = normalizer.Examples(`
-		# Joins a machine as a control plane instance
-		kubeadm join phase control-plane-join all
-		`)
```

Comment:
i think we should add this as an example for the command instead:
`Example: ....`